<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="sample.pivot.groovy.SampleIntegrationSpec" tests="0" skipped="0" failures="0" errors="0" timestamp="1970-01-01T00:00:00" hostname="testing-gce-285b8751-6141-4ab9-a69c-ed70f9132185" time="0.0">
  <properties/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[[AWT-EventQueue-0] ERROR griffon.core.GriffonExceptionHandler - Uncaught Exception. Stacktrace was sanitized. Set System property 'griffon.full.stacktrace' to 'true' for full report.
groovy.lang.MissingPropertyException: No such property: output for class: griffon.util.CompositeBuilder
	at griffon.util.CompositeBuilder$2.doCall(CompositeBuilder.java:116)
	at sample.pivot.groovy.SampleController$_sayHello_closure1.doCall(SampleController.groovy:36)
	at sample.pivot.groovy.SampleController$_sayHello_closure1.doCall(SampleController.groovy)
[AWT-EventQueue-0] ERROR griffon.core.GriffonExceptionHandler - An error occurred while handling uncaught exception groovy.lang.MissingPropertyException: No such property: output for class: griffon.util.CompositeBuilder. Stacktrace was sanitized. Set System property 'griffon.full.stacktrace' to 'true' for full report.
griffon.exceptions.InstanceNotFoundException: Could not find an instance of type griffon.core.event.EventRouter with qualifier @javax.inject.Named(value=applicationEventRouter)
	at griffon.core.GriffonExceptionHandler.handle(GriffonExceptionHandler.java:92)
	at griffon.core.GriffonExceptionHandler.uncaughtException(GriffonExceptionHandler.java:81)
Caused by: griffon.exceptions.ClosedInjectorException: Injector is closed! org.codehaus.griffon.runtime.injection.GuiceInjector@79b21178
	... 2 more
]]></system-err>
</testsuite>
